.agents/skills/tinyengine-dsl-generator/scripts/validate_page.py

```
@@ -0,0 +1,167 @@
+    if result.returncode != 0:
+        return False
+
+    return True
+
+
+def check_class_name_usage(file_path: str) -> bool:
+    """检查是否正确使用 className 而不是 class"""
+    try:
+        with open(file_path, 'r', encoding='utf-8') as f:
+            data = json.load(f)
+    except:
```

Ruff (0.15.17)

[error] Do not use bare `except`

(E722)